### PR TITLE
added listing org. users

### PR DESCRIPTION
**Pull request by stromaler**
Hello, just small addon....

### CODE REVIEW

**Comment by AurelioDeRosa** [comment]
Hi @stromaler. Thank you for your PR, we really appreciate it. However, we can't merge it unless you add to the PR some tests and documentation to the README.

**Comment by stromaler** [comment]
@AurelioDeRosa OK no probs. I do so soon.

**Comment by stromaler** [comment]
@AurelioDeRosa Sorry for delay... I added documentation and test... hope its ok now.

**Comment by stromaler** [comment]
@AurelioDeRosa  I don't understand why that tests failing,,,

**Comment by AurelioDeRosa** [comment]
Tomorrow I'll have a look at it. I'll give you some feedback.

File: github.js [hunk 1]
```diff
@@ -178,6 +178,12 @@
          this.orgs = function(cb) {
             _request('GET', '/user/orgs', null, cb);
          };
+         
+         // List members of given organization
+         // -------
+         this.orgMembers = function (orgId, cb) {
+            _request('GET', '/organizations/' + orgId + '/members', options, cb);
```
Comment (AurelioDeRosa) [inline github.js]: In passing the data to `_request` you use an `options` variable that is never declared.

File: github.js [hunk 2]
```diff
@@ -178,6 +178,12 @@
          this.orgs = function(cb) {
             _request('GET', '/user/orgs', null, cb);
          };
+         
+         // List members of given organization
+         // -------
+         this.orgMembers = function (orgId, cb) {
```
Comment (AurelioDeRosa) [inline github.js]: I think that having an API that expect an ID isn't very easy to use. As we do for other methods I think we should receive a name and do the "dirty" work on behalf of the user (retrieving the ID by the name).

File: test/test.user.js [hunk 3]
```diff
@@ -54,6 +54,13 @@ describe('Github.User', function() {
       });
    });
 
+   it('should get user.orgMembers from "dolores" company', function(done) {
```
Comment (AurelioDeRosa) [inline test/test.user.js]: I'm not sure what this dolores company is but probably we should use another one. A better fit would be the [GitHub Tools organization](https://github.com/github-tools).

**Comment by AurelioDeRosa** [comment]
Hi @stromaler. Are you still interested in this PR?

**Comment by stromaler** [comment]
@AurelioDeRosa Yes, sorry, I will complete it this week.

**Comment by clayreimann** [comment]
@stromaler This was just released in v1.2.0

**Comment by stromaler** [comment]
@clayreimann sorry for that, I totally forgot after I stopped work with this lib.

**Comment by clayreimann** [comment]
@stromaler No worries, just thought I'd let you know it landed

**Pull request closed without merging**